### PR TITLE
Update Flutter-GPU.md

### PR DESCRIPTION
**Pull request by bdero**
Remove outdated information and point to useful resources.

### CODE REVIEW

File: docs/impeller/Flutter-GPU.md [hunk 1]
```diff
@@ -1,49 +1,36 @@
...
-    ```dart
-    import 'package:flutter_gpu/gpu.dart' as gpu;
-    ```
-    Check out this [examples repository](https://github.com/bdero/flutter-gpu-examples), which includes an example of [drawing a triangle](https://github.com/bdero/flutter-gpu-examples/blob/master/lib/triangle.dart), among other things.
+Flutter GPU is a low level API for building rendering packages from scratch. Graphics programming has a steep learning curve, and it's likely that most users will opt to use a higher level rendering package rather than build there own.
 
-## Reporting bugs
+## Distribution
+
+Flutter GPU can be used on the Flutter [master channel](https://docs.flutter.dev/release/upgrade#other-channels) by
```
Comment (mit-mit) [inline docs/impeller/Flutter-GPU.md:19]: Remove ` by` from the end of that line, and add a terminating `.`

File: docs/impeller/Flutter-GPU.md [hunk 2]
```diff
@@ -1,49 +1,36 @@
-**Flutter GPU** (previously referred to as "Dart GPU" or "Impeller Dart") is an effort to expose a low level graphics API in the Flutter Framework.
+**Flutter GPU** is a low level graphics API that ships as a package in the Flutter SDK. Flutter GPU enables you to build arbitrary renderers from stratch using just Dart and GLSL. No native platform code required.
 
-Design doc: https://flutter.dev/go/impeller-dart
+> [!Warning]
+> - Flutter GPU is in an early preview state and does not guarantee API stability.
+> - Flutter GPU currently requires [Impeller to be enabled](https://docs.flutter.dev/perf/impeller#availability).
+> - Automated shader building relies on the experimental Dart "Native Assets" feature.
```
Comment (mit-mit) [inline docs/impeller/Flutter-GPU.md:6]: Consider linking "Native Assets" to https://github.com/dart-lang/sdk/issues/50565

Comment (bdero) [reply docs/impeller/Flutter-GPU.md]: Done

File: docs/impeller/Flutter-GPU.md (hunk 1 above)
Comment (bdero) [reply docs/impeller/Flutter-GPU.md]: Done

**Review by mit-mit — APPROVED** [review]

**Pull request merged**